doc/api/wdac-manifest.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
A comment in this document explaining what this is and why it is here would be helpful.